config/routes.rb

```diff
@@ -3,14 +3,14 @@
 
    get "/", to: "welcome#index"
 
-
    get '/profile', to: "users#show"
    resources :users, only: [:new, :create]
    get "/obituaries", to: "obituaries#index"
    namespace :obituaries do
      get 'covid-19', to: 'covid#index'
      get 'recent', to: 'recent#index'
    end
+   resources :obituaries, only: [:new, :create, :show]
```

Comment:
Do you want to combine this with the `get "/obituaries", to: "obituaries#index"` just to keep the routes tidy?

Reply:
Ooh yeah, that's a good idea. I'll make that change real quick.